ada_moveit/calib/default/README.md

```diff
@@ -0,0 +1,40 @@
+# Default Camera Calibration Methodology
```

Comment:
Can you link this readme from the main readme? It's not obvious where to find it.